docs/source/docker_support.rst

```diff
@@ -236,7 +236,7 @@ To stop an experiment and remove its containers from the server, run:
 .. note::
 
       When deploying to a server using docker, the experiment can save files to the directory ``/var/lib/dallinger``.
```

Comment:
Maybe you want to change that line, too?

Reply:
Actually this is still true -- people save files to `/var/lib/dallinger` and it is mapped to `dallinger-data` on the host.